extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -102,6 +118,19 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
   }
 
   /** @override */
+  handleResize(width, height) {
+    const pWidth = this.element.getAttribute('width');
+    const pHeight = this.element.getAttribute('height');
+    // We want to resize only if neither returned dimension is larger than its
+    // primary counterpart, and if at least one of the returned dimensions
+    // differ from its primary counterpart.
+    if ((width != pWidth || height != pHeight)
```

Comment:
Should we re-execute the size validation checks or verify it is a value we passed in the request?  Nothing guarantees the server will return a size we provided.  Should also consider sending a gen204 ping when we get an invalid value back.